src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/CustomDatasetSource.kt

```diff
@@ -109,12 +126,17 @@ class CustomDatasetSource private constructor(
         return DatasetCatalogue().custom(location)
     }
 
+    @Deprecated(message = "Use CustomDatasetSource.stopJira(Jira, JavaDevelopmentKit) instead")
```

Comment:
Do we need to deprecate `private` methods? Without it we can also remove the `Suppress` from current line 92

Reply:
`@Deprecate` means don't use this
So if a `private` thing should not be used, then it's a proper annotation (doesn't affect API tho, so nothing in the changelog).
Naturally, `private` things are easy to change, so by default we remove it and use the replacement. But in general, there could be situations where making the replacement is hard right now, so I guess those could stay.

Reply:
Makes sense